### PR TITLE
Improve pvpadmin command

### PR DESCRIPTION
**Pull request by VinyarionHyarmendacil**
See Issue #317

### CODE REVIEW

File: src/main/java/pvpmode/internal/server/command/PvPCommandAdmin.java [hunk 1]
```diff
@@ -135,16 +178,40 @@ public boolean isUsernameIndex (String[] args, int index)
...
+            data.setSpyingEnabled (!data.isSpyingEnabled ());
+
+            ServerChatUtils.green (sender,
+                String.format ("Spying is now %s for %s",
+                    PvPCommonUtils.getEnabledString (!data.isSpyingEnabled ()),
+                    player.getDisplayName ()));
+        }
+        else
+        {
+            ServerChatUtils.yellow (sender,
```
Comment (Mahtaran) [inline src/main/java/pvpmode/internal/server/command/PvPCommandAdmin.java]: Should we send an "An admin tried to toggle your spying setting, but it was already [off/on]" message to the player? @CraftedMods

Comment (CraftedMods) [reply src/main/java/pvpmode/internal/server/command/PvPCommandAdmin.java]: No, The change is only interesting for the player if the admin did actually change something.

**Comment by VinyarionHyarmendacil** [comment]
> Looking good so far, I'll test this of course. Could you add a two entries to the changelog?

Done. Waiting on testing and confirmation now?

**Comment by Mahtaran** [comment]
Remember to change the branch this is merged into to `development`

**Comment by Mahtaran** [comment]
Never mind, I can do that myself.

**Comment by CraftedMods** [comment]
@VinyarionHyarmendacil You've to rebase this branch to `development` before we can merte the PR

**Comment by CraftedMods** [comment]
Continues in PR #365

**Pull request closed without merging**